manage.py

```diff
@@ -251,14 +268,19 @@ def add(ctx, collections, note, name, tables_only, field_counts_option, field_li
     NOTE is your name and a description of your purpose
     """
     logger = logging.getLogger('ocdskingfisher.summarize.add')
-    logger.info('Arguments: collections=%s note=%s name=%s tables_only=%s',
-                collections, note, name, tables_only)
+    logger.info('Arguments: collections=%s note=%s name=%s tables_only=%s filter_tuple=%s',
```

Comment:
filter_tuple is the internal name, but I think `filter` will make more sense to users as it's the CLI option. That said, maybe we can name it `--where` (`where`).

Reply:
I've stuck with `filter` for now, as I think that might make more sense alongside a SQL/JSON PathLanguage option.